Oracles/chainlink-prices-usd/studio.schema.ts

```diff
@@ -0,0 +1,20 @@
+import { String, Array } from "@blockflow-labs/utils";
+
+export interface PriceDB{
+  id: string;
+  contractAddress: string;
+  name: string;
+  symbol: string;
+  decimals: number;
+  quoteCurrency: string;
+  rawPrice: string;
+  price: string;
+}
+
+export interface chainlink_pair{
```

Comment:
```suggestion
export interface chainlinkPair{
```

Reply:
did this change over the code everywhere